src/indexes.rs

```diff
@@ -619,6 +619,66 @@ impl<'a> Index<'a> {
         self.update(primary_key).await
     }
 
+    /// Get the status of a given updates in a given index.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use serde::{Serialize, Deserialize};
+    /// # use std::thread::sleep;
+    /// # use std::time::Duration;
+    /// # use meilisearch_sdk::{client::*, document, indexes::*, progress::UpdateStatus};
+    /// #
+    /// # #[derive(Debug, Serialize, Deserialize, PartialEq)]
+    /// # struct Document {
+    /// #    id: usize,
+    /// #    value: String,
+    /// #    kind: String,
+    /// # }
+    /// #
+    /// # impl document::Document for Document {
+    /// #    type UIDType = usize;
+    /// #
+    /// #    fn get_uid(&self) -> &Self::UIDType {
+    /// #        &self.id
+    /// #    }
+    /// # }
+    /// #
+    /// # futures::executor::block_on(async move {
+    /// let client = Client::new("http://localhost:7700", "masterKey");
+    /// let movies = client.get_or_create("movies_get_one_update").await.unwrap();
+    ///
+    /// let progress = movies.add_documents(&[
+    ///     Document { id: 0, kind: "title".into(), value: "The Social Network".to_string() },
+    ///     Document { id: 1, kind: "title".into(), value: "Harry Potter and the Sorcerer's Stone".to_string() },
+    /// ], None).await.unwrap();
+    ///
+    /// let update_id = progress.get_update_id();
+    /// let status = movies.get_update(update_id).await.unwrap();
```

Comment:
I fear that some users will misunderstand this example. They might think that this is the only way to get a `Progress` and they might use this instead of `progress.get_status()`. Could you make this clearer? By telling that they should use `progress.get_status()` in most cases and explaining when this is intended to be used.

Reply:
I did a couple of changes to try and tackle this.

First in this very function description, I added a sentence that describes the normal course of action, that is, getting the status directly from the progress object.

Second, in the code example, I obtain the status in both ways, from the progress object, and by calling `get_update` on the index with the `update_id`.

I also modified (and fixed after my last commit) the `.code-samples.meilisearch.yml` file, by expanding the `get_update_md` section (formerly `get_update_1`), with code for both use cases.

I am not so good with copy or technical writing, so I was forced to introduce the wording **"progress object"** which might not be ideal.

What do you think? Does it look good now?

Reply:
Great! I like this documentation.

> I also modified (and fixed after my last commit) the .code-samples.meilisearch.yml file, by expanding the get_update_md section (formerly get_update_1), with code for both use cases.

That would be perfect if it worked but I am not sure that the documentation scraper is powerful enough. Will this work @curquiza ? Is the format of the documentation automatically detected from the field name suffix or is it hardcoded ? 

> I am not so good with copy or technical writing, so I was forced to introduce the wording "progress object" which might not be ideal.

This is indeed not ideal. I would rather say **"`Progress` struct"**. But the second occurrence of "object" is fine in my opinion.

Reply:
Hello! It will indeed not work in the docs, but you can change it like this: https://github.com/meilisearch/meilisearch-rust/pull/133/files#r638115933